science/2022-11-23-science-blog-6/index_de.md

```diff
@@ -0,0 +1,237 @@
+---
+page-title: "Was sagt eine rote Warnung bei hohen Inzidenzen noch aus?"
+page-description: "Was sagt eine rote Warnung bei hohen Inzidenzen noch aus?"
+page-name: science-blog-6
+page-name_de: science-blog-6
+author: CWA-Team
+date: "2022-11-23"
+layout: science
+---
+
+In diesem Beitrag wollen wir untersuchen, welche Aussagekraft eine rote Warnung gegenüber einem niedrigen Risiko mit bzw. ohne Risikobegegnung hat. Hierzu betrachten wir den Positivenanteil der Testergebnisse, d.h. den Anteil der positiven Testergebnisse, in Abhängigkeit vom angezeigten Risiko. Die Daten hierfür stammen aus der [Datenspende der Corona-Warn-App](../2021-10-15-science-blog-4/#3-privacy-preserving-analytics).
```

Comment:
The link [Datenspende der Corona-Warn-App](../2021-10-15-science-blog-4/#3-privacy-preserving-analytics) fails, since it is resolved to `/de/2021-10-15-science-blog-4/`

Change to `/de/science/2021-10-15-science-blog-4/#3-privacy-preserving-analytics`

See also advice in https://github.com/corona-warn-app/cwa-website/blob/master/docs/WRITE_BLOG_POSTS.md#link-to-other-locations which also applies to using `../` type links in science blogs.